recipe/meta.yaml

```diff
@@ -17,53 +17,54 @@ build:
 outputs:
   - name: keras-base
     script: install_base.sh  # [not win]
-    script: install_base.bat # [win]
+    script: install_base.bat  # [win]
     build:
       noarch: python
     requirements:
       host:
         - python >=3.6
         - pip
+        - setuptools
+        - wheel
       run:
         - python >=3.6
-        # - numpy >=1.9.1
-        # - scipy >=0.14
-        # - pyyaml
-        # - h5py
-        # - tensorflow >=2.2
+        - numpy >=1.9.1
+        - scipy >=0.14
+        - pyyaml
+        - h5py
+        - tensorflow  
       # add run_constrained to prevent installation of mismatched keras and
       # keras-base packages
       run_constrained:
-        # - keras {{ version }}
-        - tensorflow {{ version }}
+        - keras {{ version }}
     test:
       requires:
         - pip
       commands:
         - pip check
-      # imports:
-        # - keras
-        # - keras.backend
-        # - keras.datasets
-        # - keras.engine
-        # - keras.layers
-        # - keras.preprocessing
-        # - keras.utils
-        # - keras.wrappers
-
+      imports:
+        - keras
+        - keras.backend
+        - keras.datasets
+        - keras.engine
+        - keras.layers
+        - keras.preprocessing
+        - keras.utils
+        - keras.wrappers
+ #some of the imports are commented out during testing to avoid circular dependencies
   - name: keras
     script: install_base.sh  # [not win]
-    script: install_base.bat # [win]
+    script: install_base.bat  # [win]
     build:
       noarch: true
     requirements:
       host:
         - python >=3.6
         - pip
+        - setuptools
+        - wheel
       run:
         - python >=3.6
-      run_constrained:
-        - tensorflow {{ version }}
```

Comment:
this constrain should stay

Reply:
The constrains for tensorflow in the run_constrained were restored:
https://github.com/AnacondaRecipes/keras-feedstock/blob/update_to_2.6.0/recipe/meta.yaml#L68&L69